### PR TITLE
Release v0.39.0

### PR DESCRIPTION
**Pull request by kachawla**
# Description

* Adding 0.39.0 to the supported versions
* Deprecating v0.38.0

## Type of change

- This pull request is a minor refactor, code cleanup, test improvement, or other maintenance task and doesn't change the functionality of Radius (issue link optional).

Fixes: #issue_number

## Contributor checklist
Please verify that the PR meets the following requirements, where applicable:

- [ ] An overview of proposed schema changes is included in a linked GitHub issue. -- N/A
- [ ] A design document PR is created in the [design-notes repository](https://github.com/radius-project/design-notes/), if new APIs are being introduced. -- N/A
- [ ] If applicable, design document has been reviewed and approved by Radius maintainers/approvers. -- N/A
- [ ] A PR for the [samples repository](https://github.com/radius-project/samples) is created, if existing samples are affected by the changes in this PR. -- N/A
- [ ] A PR for the [documentation repository](https://github.com/radius-project/docs) is created, if the changes in this PR affect the documentation or any user facing updates are made. -- N/A
- [ ] A PR for the [recipes repository](https://github.com/radius-project/recipes) is created, if existing recipes are affected by the changes in this PR. -- N/A

### CODE REVIEW

**Review by ytimocin — DISMISSED** [review]

File: docs/release-notes/v0.39.0.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+https://github.com/radius-project/radius/pull/7906
+
+### Add an in-memory data store
+https://github.com/radius-project/radius/pull/7950
+
+### Implement PostgreSQL datastore
+https://github.com/radius-project/radius/pull/7961
+
+## Breaking changes
+https://github.com/radius-project/radius/pull/7906
```
Comment (kachawla) [inline docs/release-notes/v0.39.0.md:23]: @willtsai could you please help updating these. I have copied over PRs of a few items from changelog that might be good candidates for highlights.

Comment (willtsai) [inline docs/release-notes/v0.39.0.md:23]: I think `Implement PostgreSQL datastore` is too early of an addition to call out in highlights, we should call it out when it's available for users to switch to a postgres datastore.

I don't think this is a breaking change, so I removed it: https://github.com/radius-project/radius/pull/7906

```suggestion
## Highlights

### Support for Dapr Configuration Building Block
A new `Applications.Dapr/configurationStores` resource type is now available, which enables built-in support for the Dapr [Configuration](https://docs.dapr.io/getting-started/quickstarts/configuration-quickstart/) building block in Radius. The Dapr Configuration building block allows for dynamic configuration updates, including feature flag management, and is a great addition to the set of supported Dapr features in Radius. To learn more, go to the [Dapr Configuration Store resource schema](https://docs.radapp.io/reference/resource-schema/dapr-schema/configurationstore/) in the Radius documentation.

### In-memory data store for testing and development
Radius has added an implementation for an in-memory version of the data store interface, which may be used for testing and development purposes. This helps to speed up and simplify the existing ETC.d and mocking mechanisms that are frequently used today in tests.

## Breaking changes
None.
```

File: docs/release-notes/v0.39.0.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+We would like to extend our thanks to all the [new](#new-contributors) and existing contributors who helped make this release possible!
+
+## Intro to Radius
+
+If you're new to Radius, check out our website, [radapp.io](https://radapp.io), for more information. Also visit our [getting started guide](https://docs.radapp.io/getting-started/) to learn how to install Radius and create your first app.
+
+## Highlights
+
+### Support for Dapr Configuration Building Block
+A new `Applications.Dapr/configurationStores` resource type is now available, which enables built-in support for the Dapr [Configuration](https://docs.dapr.io/getting-started/quickstarts/configuration-quickstart/) building block in Radius. The Dapr Configuration building block allows for dynamic configuration updates, including feature flag management, and is a great addition to the set of supported Dapr features in Radius. To learn more, go to the [Dapr Configuration Store resource schema](https://docs.radapp.io/reference/resource-schema/dapr-schema/configurationstore/) in the Radius documentation.
```
Comment (willtsai) [inline docs/release-notes/v0.39.0.md:14]:
```suggestion
A new `Applications.Dapr/configurationStores` resource type is now available, which adds built-in support in Radius for the [Dapr Configuration](https://docs.dapr.io/getting-started/quickstarts/configuration-quickstart/) building block in Radius. The Dapr Configuration building block allows for dynamic configuration updates, including feature flag management, and is a great addition to the set of supported Dapr features in Radius. To learn more, go to the [Dapr Configuration Store resource schema](https://docs.radapp.io/reference/resource-schema/dapr-schema/configurationstore/) in the Radius documentation.
```

**Review by willtsai — DISMISSED** [review]

File: docs/release-notes/v0.39.0.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+## Highlights
+
+### Support for Dapr Configuration Building Block
+A new `Applications.Dapr/configurationStores` resource type is now available, which adds built-in support in Radius for the [Dapr Configuration](https://docs.dapr.io/getting-started/quickstarts/configuration-quickstart/) building block in Radius. The Dapr Configuration building block allows for dynamic configuration updates, including feature flag management, and is a great addition to the set of supported Dapr features in Radius. To learn more, go to the [Dapr Configuration Store resource schema](https://docs.radapp.io/reference/resource-schema/dapr-schema/configurationstore/) in the Radius documentation.
+
+### In-memory data store for testing and development
+Radius has added an implementation for an in-memory version of the data store interface, which may be used for testing and development purposes. This helps to speed up and simplify the existing ETC.d and mocking mechanisms that are frequently used today in tests.
```
Comment (rynowak) [inline docs/release-notes/v0.39.0.md:17]: We should make it really clear that this is for contributors not users.

Comment (kachawla) [reply docs/release-notes/v0.39.0.md:17]: `Radius has added an implementation for an in-memory version of the data store interface, which may be used for testing and development purposes, when contributing to Radius.`

Does this addition at the end make it clear @rynowak? @willtsai please update if you have thoughts on this.

Comment (rynowak) [reply docs/release-notes/v0.39.0.md:17]: I think that's good enough 👍

**Review by ytimocin — DISMISSED** [review]

File: docs/release-notes/v0.39.0.md (hunk 3 above)
Comment (kachawla) [inline docs/release-notes/v0.39.0.md:17]:
```suggestion
Radius has added an implementation for an in-memory version of the data store interface, which may be used for testing and development purposes, when contributing to Radius. This helps to speed up and simplify the existing ETC.d and mocking mechanisms that are frequently used today in tests.
```

File: docs/release-notes/v0.39.0.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+We would like to extend our thanks to all the [new](#new-contributors) and existing contributors who helped make this release possible!
+
+## Intro to Radius
+
+If you're new to Radius, check out our website, [radapp.io](https://radapp.io), for more information. Also visit our [getting started guide](https://docs.radapp.io/getting-started/) to learn how to install Radius and create your first app.
+
+## Highlights
+
+### Support for Dapr Configuration Building Block
+A new `Applications.Dapr/configurationStores` resource type is now available, which adds built-in support in Radius for the [Dapr Configuration](https://docs.dapr.io/getting-started/quickstarts/configuration-quickstart/) building block in Radius. The Dapr Configuration building block allows for dynamic configuration updates, including feature flag management, and is a great addition to the set of supported Dapr features in Radius. To learn more, go to the [Dapr Configuration Store resource schema](https://docs.radapp.io/reference/resource-schema/dapr-schema/configurationstore/) in the Radius documentation.
```
Comment (Reshrahim) [inline docs/release-notes/v0.39.0.md:14]:
```suggestion
A new `Applications.Dapr/configurationStores` resource type is now available, to define and deploy [Dapr Configuration](https://docs.dapr.io/getting-started/quickstarts/configuration-quickstart/) building block in Radius. The Dapr Configuration building block allows for dynamic configuration updates, including feature flag management, and is a great addition to Dapr integration in Radius. To learn more, go to the [Dapr Configuration Store resource schema](https://docs.radapp.io/reference/resource-schema/dapr-schema/configurationstore/) in the Radius documentation. Thanks to community contributor @SoTrx for adding this functionality.
```

**Review by ytimocin — DISMISSED** [review]

**Review by ytimocin — DISMISSED** [review]

**Review by Reshrahim — APPROVED** [review]

File: docs/release-notes/v0.39.0.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+We would like to extend our thanks to all the [new](#new-contributors) and existing contributors who helped make this release possible!
+
+## Intro to Radius
+
+If you're new to Radius, check out our website, [radapp.io](https://radapp.io), for more information. Also visit our [getting started guide](https://docs.radapp.io/getting-started/) to learn how to install Radius and create your first app.
+
+## Highlights
+
+### Support for Dapr Configuration Building Block
+A new `Applications.Dapr/configurationStores` resource type is now available, to define and deploy [Dapr Configuration](https://docs.dapr.io/getting-started/quickstarts/configuration-quickstart/) building block in Radius. The Dapr Configuration building block allows for dynamic configuration updates, including feature flag management, and is a great addition to the Dapr integration in Radius. To learn more, go to the [Dapr Configuration Store resource schema](https://docs.radapp.io/reference/resource-schema/dapr-schema/configurationstore/) in the Radius documentation. Thanks to community contributor @SoTrx for adding this functionality.
```
Comment (willtsai) [inline docs/release-notes/v0.39.0.md:14]:
```suggestion
A new `Applications.Dapr/configurationStores` resource type is now available to define and deploy the [Dapr Configuration](https://docs.dapr.io/getting-started/quickstarts/configuration-quickstart/) building block in Radius. The Dapr Configuration building block allows for dynamic configuration updates, including feature flag management, and is a great addition to the Dapr integration in Radius. To learn more, go to the [Dapr Configuration Store resource schema](https://docs.radapp.io/reference/resource-schema/dapr-schema/configurationstore/) in the Radius documentation. Thanks to community contributor @SoTrx for adding this functionality.
```

**Review by willtsai — APPROVED** [review]

**Review by ytimocin — APPROVED** [review]

**Pull request merged**